packages/react-select/src/components/Select/useSelectStyles.ts

```diff
@@ -12,36 +13,211 @@ export const selectClassNames: SlotClassNames<SelectSlots> = {
   icon: 'fui-Select__icon',
 };
 
-const useStyles = makeStyles({
-  wrapper: {
-    // TODO: add styles
+// TODO(sharing) use theme values once available
+const horizontalSpacing = {
+  xxs: '2px',
+  xs: '4px',
+  sNudge: '6px',
+  s: '8px',
+  mNudge: '10px',
+  m: '12px',
+};
+const contentSizes = {
+  // TODO: borrowed this from Input, should be in the theme somewhere?
+  body1: {
+    fontSize: tokens.fontSizeBase300,
+    lineHeight: tokens.lineHeightBase300,
+  },
+  caption1: {
+    fontSize: tokens.fontSizeBase200,
+    lineHeight: tokens.lineHeightBase200,
+  },
+  400: {
+    fontSize: tokens.fontSizeBase400,
+    lineHeight: tokens.lineHeightBase400,
+  },
+};
+// TODO: borrowed this from Input (Select has the same size in design comps)
+// Should be in the theme somewhere?
+const fieldHeights = {
+  small: '24px',
+  medium: '32px',
+  large: '40px',
+};
+
+const iconSizes = {
+  small: '16px',
+  medium: '20px',
+  large: '24px',
+};
+
+// TODO: borrowed from Input, also seems like animation should be included in the theme:
+const motionDurations = {
+  ultraFast: '0.05s',
+  normal: '0.2s',
+};
+const motionCurves = {
+  accelerateMid: 'cubic-bezier(0.7,0,1,0.5)',
+  decelerateMid: 'cubic-bezier(0.1,0.9,0.2,1)',
+};
```

Comment:
```suggestion
```

you only use each of these values once, so you might not need to have a separate object for it

Reply:
The reason I pulled them out is b/c they're also used in Input styles, and I think it'd be good to find a way in the future to share a bunch of these values with the input package (and in the future other components like spinbutton, maybe textarea, etc).

I have a mild preference for keeping it separate so all the shared values are in one place, but I agree that if these values continue being defined in `useSelectStyles`, it'd make sense to just put them inline.

I'll restructure the shared values slightly and put a bigger `TODO` comment above all of them with more explanation -- how does that sound to you for now?

Reply:
Can we track this in an issue instead and add it to the react-components board, TODOs in the code will always be lost

Reply:
it seems that the styles have found a home in `react-tabster` in #22096 thanks to @sopranopillow

Reply:
issue created!

Reply:
`react-tabster` will be its home for now, we'll probably decide where it should live on wednesday.